src/expertsystem/reaction/__init__.py

```diff
@@ -483,7 +481,7 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-branches
             )
 
         if reload_pdg or len(self.__particles) == 0:
-            self.__particles = load_default_particles()
+            self.__particles = load_pdg()
```

Comment:
Isn't the full default particle list needed here? So pdg + the additional particle definitions?

Reply:
Can't import from the top module in the `reaction` module:

> load_default_particles had to be moved to the top import, so that the reaction module is not dependent on the io module (the type hint for Result in the io introduce a circular dependency).

Reply:
So the default has changed slightly: (only) this particle is not provided anymore by default
https://github.com/ComPWA/expertsystem/blob/1e8b81773342b022cdfe41d18d8d6e4c6e540dcf/src/expertsystem/particle/additional_definitions.yml#L2-L13

Reply:
Ok